docs/governance/staking-with-realms.md

```diff
@@ -155,32 +142,41 @@ the overall lockup multiplier.
   </figcaption>
 </figure>
 
+### Delegating to a subDAO (veHNT Only)
+
+Once you select how much HNT you want to lock up, an interface will appear to allow you to delegate 
+your veHNT to a subDAO for rewards. A stake will not earn any token return until it is delegated to 
+a subDAO. A stake without delegation earns voting rights, but no token return. You can delegate an 
+active stake within the Helium Vote interface.
+
+<figure className="screensnippet-wrapper">
+  <img src={useBaseUrl('https://github.com/helium/docs/pull/1784#issuecomment-1933068839')} style={{ maxHeight: 300 }} />
+  <figcaption>
+    You may choose to delegate your stake to either the MOBILE or IOT subDAO.
+  </figcaption>
+</figure>
+
 ### Confirming your Stake
 
-Once parameters are settled in the Realms UI, you are now ready to create your veHNT position.
+Once parameters are settled in the Helium Vote UI, you are now ready to create your veHNT position.
 
 The Helium Wallet App will ask for confirmation before issuing the Solana transaction that creates
-the veHNT position.
+the veHNT/veMOBILE/veIOT position.
 
 <figure className="screensnippet-wrapper">
-  <img src={useBaseUrl('/img/realms/issuestake.png')} style={{ maxHeight: 300 }} />
+  <img src={useBaseUrl('https://github.com/helium/docs/pull/1784#issuecomment-1933070813')} style={{ maxHeight: 300 }} />
   <figcaption>
-    After pressing, "Lock Tokens", a Solana transaction is issued to create the stake.
+    After pressing "Lock Tokens", a Solana transaction is issued to create the stake.
   </figcaption>
 </figure>
 
 ## Managing Stakes
```

Comment:
We should note that any changes to delegations will forego the delegation rewards for that day. e.g. moving from mobile to iot

Reply:
This is already mentioned in the Undelegating section

_Once a stake has been delegated, it can be undelegated in the same interface. Delegations can be
changed at any time, however, it will take one epoch for the position to be available for
redelegation. The stake must be delegated for the entire epoch to receive rewards for that epoch.
Received rewards need to be claimed manually. Only then will they be sent to the wallet the veHNT
position is in. A delegated position can only be undelegated after all accrued rewards have been
claimed._